eva_cttv_pipeline/validations/validation_trait_mapping_pipeline.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+clinvar_filtered=$1
+automated=$2
+manual=$3
+
+if [ -z "$clinvar_filtered" ] || [ -z "$automated" ] || [ -z "$manual" ]
+then
+	echo "Please provide all 3 parameters [clinvar.filtered.json.gz] [files automated_trait_mappings.tsv] [traits_requiring_curation.tsv]"
+	exit 1
+fi
+
+##PROCESS JSON FILE
+
+#uncompress file
+gunzip -k ${clinvar_filtered}
+
+#extracts only the value type (Preferred or Alternate) and the trait name
+jq .clinvarSet.referenceClinVarAssertion.traitSet.trait[].name[].elementValue clinvar.filtered.json > trait_names.txt
+
+#extracts only the trait names which are preferred
+grep -B 1 "Preferred" trait_names.txt > trait_names_preferred.txt
+
+#if the line contains the phrase between the forward slashes, the whole line is deleted
```

Comment:
All the sed commands below can be made a single rule to reduce the amount of disk writes, see [this post](https://unix.stackexchange.com/a/174613).

The applied changes can be documented in a longer comment.